code/modules/mob/language/station.dm

```diff
@@ -179,6 +179,17 @@
 	syllables = list("qr","qrr","xuq","qil","quum","xuqm","vol","xrim","zaoo","qu-uu","qix","qoo","zix","*","!")
 	allow_accents = TRUE
 
+/datum/language/skrell/check_speech_restrict(mob/speaker)
+	if(!ishuman(speaker))
+		return FALSE
+	var/mob/living/carbon/human/H = speaker
+	var/obj/item/organ/internal/augment/language/zeng/aug = H.internal_organs_by_name[BP_AUG_LANGUAGE]
+	if(istype(aug))
+		to_chat(speaker, SPAN_WARNING("You are not capable of speaking Nral'malic!"))
+		return FALSE
```

Comment:
Wouldn't this prevent a skrell without the augment from using nral'malic?

Reply:
This prevents someone with the augment from speaking it. Though Skrell can't normally take it, I should add a check to correct cases like that actually.

Reply:
Added a check so that giving a Skrell the implant will not delete its vocal cords